detoks-uvt/src/main/java/nl/tudelft/trustchain/detoks/TorrentManager.kt

```diff
@@ -191,7 +204,22 @@ class TorrentManager(
         fileOrDirectory.delete()
     }
 
-    class TorrentHandler(
+    class TorrentHandlerPlusUserInfo(cacheDir: File,
```

Comment:
Do you guys think that the name of this class is okay? If not, what would be a better name for this class?

Reply:
I think that this is fine for now as it dependent on another groups work, so it will be changed later anyway